sdk/Sdk/Tasks/EnhanceExtensionsMetadata.cs

```diff
@@ -24,12 +25,20 @@ public class EnhanceExtensionsMetadata : Task
         [Required]
         public string? OutputPath { get; set; }
 
+        [Required]
```

Comment:
Is this a breaking change to introduce a required field to a public class?

Reply:
I am not sure I consider this public API. It is a supporting task for our build targets, not expected to be used directly by customers.